api/endpoints/room.md

```diff
@@ -73,11 +73,15 @@ Collection of all Rooms.
 + Parameters
```

Comment:
Should the `## Room Collection...` line above be updated with the new params too?